packages/SwingSet/src/kernel/vatTranslator.js

```diff
@@ -162,12 +162,12 @@ function makeTranslateVatSyscallToKernelSyscall(vatID, kernelKeeper) {
   }
 
   function translateSubscribe(promiseID) {
-    const id = mapVatSlotToKernelSlot(promiseID);
-    kdebug(`syscall[${vatID}].subscribe(vat:${promiseID}=ker:${id})`);
-    if (!kernelKeeper.hasKernelPromise(id)) {
-      throw new Error(`unknown kernelPromise id '${id}'`);
+    const kpid = mapVatSlotToKernelSlot(promiseID);
```

Comment:
good, and let's also change that `promiseID` to `vpid`

The only problem I've had with using `k|v` plus `p|o|d` plus `id` is that vat object ids wind up wanting to live in a variable named `void`, which won't work. So I think I've gone non-uniform wth `vobjid` or `vObjectID` or something for that one outlier. Fortunately there aren't many places where we have an ID which is known to be vat-specific and also known to reference an object, so it doesn't come up very often.

When we've got a reference that is known to be vat-specific but we don't know what type it holds, maybe `vid` (and matching `kid`) could work, and we just assume that `vatID` (naming a vat) is clearly distinct from `vid` (naming some unknown vat's object/promise/device reference).